lod/instruction-source.md

```diff
@@ -48,6 +49,16 @@ These include:
 
 1. **Activities** - Automated and question activities are unavailable when using an external repository as an instruction source. 
 
+## Best Practices
+ 
+ When using an external instruction source, there are a few things to consider. 
+
+ - The overall size of the repository cannot exceed 500MB.
```

Comment:
We may need to explicitly state that the 500mb limit includes commit history if shallow checkout isn't going to be an option.